services/user.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.auth import get_user_model
+from typing import Optional
+
+User = get_user_model()

```

Comment:
According to the checklist, it's better to use `get_user_model()` directly instead of assigning it to the `User` variable. This is a best practice in Django to avoid potential issues with custom user models.